webapp/src/components/App/Singlestudy/index.tsx

```diff
@@ -102,6 +103,7 @@ function SingleStudy(props: Props) {
     const init = async () => {
       if (studyId) {
         dispatch(setCurrentStudy(studyId));
+        dispatch(setDefaultAreaLinkSelection(studyId));
```

Comment:
Include it in `setCurrentStudy`: transform `setCurrentStudy` into a thunk and call `setCurrentArea` inside